test/evergreen/run_format_configs.sh

```diff
@@ -3,8 +3,7 @@
 # Cycle through a list of test/format configurations that failed previously,
 # and run format test against each of those configurations to capture issues.
 #
-
-set -u
+# Note - Any of the failed configs does not require a restart.
```

Comment:
Suggested wording "Note - The failed configs must not require restart (such as abort)."

Reply:
Fixed in 107178c66c6a6792139b32ebc6fd44432ba6b88f